test/integration/e2e/admin-api/postUsers.test.ts

```diff
@@ -43,10 +44,54 @@ describe('E2E - adminApiRouter - POST /users', function (): void {
       .expect('Content-Type', /text\/plain/u)
       // THEN we expect the Location header to be set to the path of the created user resource
       .expect('Location', `/users/${userInformation.payId}`)
-      // AND we expect back a 201 - CREATED
+      // THEN we expect back a 201 - CREATED
       .expect(HttpStatus.Created, done)
   })
 
+  it('Returns a 201 when creating a new user, without an Accept-Patch header in the response', function (done): void {
+    // GIVEN a user with a PayID known to not exist on the PayID service
+    const userInformation = {
+      payId: 'johnfoo$xpring.money',
+      addresses: [
+        {
+          paymentNetwork: 'XRPL',
+          environment: 'TESTNET',
+          details: {
+            address: 'TVQWr6BhgBLW2jbFyqqufgq8T9eN7KresB684ZSHKQ3oDtq',
+          },
+        },
+        {
+          paymentNetwork: 'BTC',
+          environment: 'TESTNET',
+          details: {
+            address: 'mxNEbRXokcdJtT6sbukr1CTGVx8Tkxk3DC',
+          },
+        },
+      ],
+    }
+
+    // WHEN we make a POST request to /users with that user information
+    request(app.adminApiExpress)
+      .post(`/users`)
+      .set('PayID-API-Version', payIdApiVersion)
+      .send(userInformation)
+      .expect('Content-Type', /text\/plain/u)
+      // THEN we expect the Location header to be set to the path of the created user resource
+      .expect('Location', `/users/${userInformation.payId}`)
+      // THEN we expect back a 201 - CREATED
+      .expect(HttpStatus.Created)
+      .end(function (err, res) {
+        if (err) {
+          return done(err)
+        }
+
+        // AND ensure Accept-Patch header does not exist
```

Comment:
Should this be `AND` or another `THEN`? Providing the suggestion just in case.

```suggestion
        // THEN ensure Accept-Patch header does not exist
```